pkg/service/submission/storage/bolt.go

```diff
@@ -0,0 +1,63 @@
+package storage
+
+import (
+	"fmt"
+
+	"github.com/goccy/go-json"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+)
+
+const (
+	namespace = "presentation_submission"
+)
+
+type BoltSubmissionStorage struct {
```

Comment:
think this should be merged into the presentation storage

Reply:
Done.